release/test_install.py

```diff
@@ -6,7 +6,7 @@
 from subprocess import check_call
 
 
-PY_VERSIONS = '3.7', '3.8', '3.9'
+PY_VERSIONS = '3.8', '3.9'
```

Comment:
I'm wondering if 3.10 should be added here now? @oscarbenjamin can probably tell.

Reply:
The different python versions used here is provided by `release/aptinstall.sh`. According to https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa/+packages, they do provide python 3.10 packages on ubuntu 20.04. So I think this could be added.

Reply:
Yes, 3.10 should be added here.